server/project.go

```diff
@@ -160,7 +160,11 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 		}
 
 		repositoryCreate.WebhookURLHost = fmt.Sprintf("%s:%d", s.host, s.port)
-		repositoryCreate.WebhookEndpointId = uuid.NewV4().String()
+		v4, err := uuid.NewRandom()
```

Comment:
By looking at google's package. I would rather just use uuid.New https://pkg.go.dev/github.com/google/uuid#New

If this method returns an error from the random reader https://github.com/google/uuid/blob/655bf50db9d265813a26b0fe2a5d1e80fb9e3c6b/uuid.go#L40

Then the shit really hits the fan and it's Ok to just panic.

What do you think?

Reply:
It's ok, RecoverMiddleware will recover.